ui/src/containers/NodeVolumes.js

```diff
@@ -95,11 +229,47 @@ const NodeVolumes = props => {
     volumeSortedList = sortSelector(volumeSortedList, sortBy, sortDirection);
   }
 
+  const onClickDeleteButton = deleteVolumeName => {
+    props.deleteVolume(deleteVolumeName);
+    setisDeleteConfirmationModalOpen(false);
+  };
+
+  const onClickCancelButton = () => {
+    setisDeleteConfirmationModalOpen(false);
+  };
+
   return (
     <>
+      <Modal
+        isOpen={isDeleteConfirmationModalOpen}
+        ariaHideApp={false}
+        style={DeleteConfirmationModal}
+      >
+        <h2>{intl.messages.delete_a_volume}</h2>
+        <div>{intl.messages.delete_a_volume_warning}</div>
+        <div>
+          {intl.messages.delete_a_volume_confirm}{' '}
```

Comment:
Should you pass ```deleteVolumeName``` as ```delete_a_volume_confirm``` 's argument

Reply:
The reason I didn't pass `deleteVolumeName` inside the translation is that I would like to give bold to the deleteVolumeName in the Modal. So I will keep in this way ^^